cluster-api/machines-api/types.go

```diff
@@ -0,0 +1,160 @@
+package machines
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kubernetes/pkg/api"
```

Comment:
This can be removed and we can use `k8s.io/api/core/v1` instead.

Reply:
Yup, this was an accident that got fixed in the merged types but not this PR. I'll update the PR with the newest definition from the codebase.